src/vivarium_public_health/population/base_population.py

```diff
@@ -153,9 +152,11 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
 
     def on_time_step(self, event: Event) -> None:
         """Ages simulants each time step."""
-        population = self.population_view.get(event.index, query="alive == 'alive'")
-        population["age"] += utilities.to_years(event.step_size)
-        self.population_view.update(population)
+        age = self.population_view.get_private_columns(
+            event.index, ["age"], query_columns=["alive"], query="alive == 'alive'"
```

Comment:
In some other methods you've enabled the user to pass a string if they only want one column. Should we do this for `get_private_columns()`? 

Is it a stupid idea to have multiple methods on `PopulationView` - `get_attribute()` and `get_private_column()` which take a string and return a Series, and `get_attributes()` and `get_private_columns()` which take a list of strings and return a DataFrame?

Reply:
I don't personally like having different methods to handle plurality but happy to talk about it more.